cmd/uhc/get/cmd.go

```diff
@@ -78,112 +48,14 @@ func run(cmd *cobra.Command, argv []string) {
 		os.Exit(1)
 	}
 
-	// Load the configuration file:
-	cfg, err := config.Load()
-	if err != nil {
-		fmt.Fprintf(os.Stderr, "Can't load config file: %v\n", err)
-		os.Exit(1)
-	}
-	if cfg == nil {
-		fmt.Fprintf(os.Stderr, "Not logged in, run the 'login' command\n")
-		os.Exit(1)
-	}
-
-	// Check that the configuration has credentials or tokens that don't have expired:
-	armed, err := config.Armed(cfg)
-	if err != nil {
-		fmt.Fprintf(os.Stderr, "Can't check if tokens have expired: %v\n", err)
-		os.Exit(1)
-	}
-	if !armed {
-		fmt.Fprintf(os.Stderr, "Tokens have expired, run the 'login' command\n")
-		os.Exit(1)
-	}
-
-	// Create the connection:
-	logger, err := util.NewLogger(args.debug)
-	if err != nil {
-		fmt.Fprintf(os.Stderr, "Can't create logger: %v\n", err)
-		os.Exit(1)
-	}
-	connection, err := client.NewConnectionBuilder().
-		Logger(logger).
-		TokenURL(cfg.TokenURL).
-		Client(cfg.ClientID, cfg.ClientSecret).
-		Scopes(cfg.Scopes...).
-		URL(cfg.URL).
-		User(cfg.User, cfg.Password).
-		Tokens(cfg.AccessToken, cfg.RefreshToken).
-		Insecure(cfg.Insecure).
-		Build()
+	connection, err := util.NewConnection(args.Debug)
```

Comment:
I understand that there is a lot of code duplication here. But I am a bit concerned about moving all this logic to the `NewConnection` method. Specially the user interface logic: generating and printing the messages to the user. Also not all commands need all that logic. The `config get` and `config set` commands only need to load/save the configuration, but they don't need a connection. Can we move to that new method only the part that creates the connection? And make it a method of the `Config` type? This would then be:

```go
// Create the connection:
connect, err := cfg.Connection()
if err != nil {
        ...
}
```

Inside that `Connection` method there should be as little user interface code as possible, and no `fmt.Fprintf(...)` or `os.Exit(...)` calls.